webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/Virtualization/VirtOvirt.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+} from '@patternfly/react-core';
+import { number_to_human_size as NumberToHumanSize } from 'number_helpers';
+import { translate as __ } from '../../../../../../common/I18n';
+
+const VirtOvirt = ({ vm }) => (
+  <>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Name')}</DescriptionListTerm>
+      <DescriptionListDescription>{vm.name}</DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Cores per socket')}</DescriptionListTerm>
+      <DescriptionListDescription>{vm.cores}</DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Number of sockets')}</DescriptionListTerm>
+      <DescriptionListDescription>{vm.sockets}</DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Memory')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        {NumberToHumanSize(vm.memory, { strip_insignificant_zeros: true })}
+      </DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Display')}</DescriptionListTerm>
+      <DescriptionListDescription>{vm.display.type}</DescriptionListDescription>
+    </DescriptionListGroup>
+    {vm.display.type === 'vnc' && (
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Keyboard')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {vm.display.keyboard_layout}
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+    )}
+    {Object.values(vm.interfaces_attributes).map((nic, index) => (
+      <DescriptionListGroup key={index}>
+        <DescriptionListTerm>{__('NIC name')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {nic.compute_attributes.name}
+        </DescriptionListDescription>
+        <DescriptionListTerm>{__('Network')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {nic.compute_attributes.network}
+        </DescriptionListDescription>
+        <DescriptionListTerm>{__('MAC')}</DescriptionListTerm>
+        <DescriptionListDescription>{nic.mac}</DescriptionListDescription>
+      </DescriptionListGroup>
+    ))}
+    {Object.values(vm.volumes_attributes).map((vol, index) => (
+      <DescriptionListGroup key={index}>
```

Comment:
This will have the same indexes as the interfaces while `key` implies it should be unique. Will that be a problem? If not, why do you even have a key?